nettracersrv/nettracer.cpp

```diff
@@ -122,6 +134,12 @@ bool isIPv6MonitoringPossible(int status_fd, bpf::BPFMapsWrapper& mapsWrapper) {
 	return mapsWrapper.lookupElement(status_fd, &zero, &status) && status.offset_daddr_ipv6 != 0;
 }
 
+enum ReturnCodes {
```

Comment:
Although There is no standard for exit codes there is file /usr/include/sysexits.h

Reply:
Alright, so you propose to use EX_OK, EX_NO_PERM and EX_SOFTWARE? That could be ok, though I was thinking about extending the codes later in the future in a way specific to NetTracer, e.g. with offset guessing errors.